lib/AcqUnitFilter/AcqUnitFilter.js

```diff
@@ -1,11 +1,14 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
 
 import { SelectionFilter } from '../SelectionFilter';
 import { getAcqUnitsOptions } from '../utils';
-import { acqUnitsShape } from '../shapes';
+import { acqUnitsResource } from '../manifests';
 
-const AcqUnitFilter = ({ acqUnits, ...rest }) => {
-  const options = getAcqUnitsOptions(acqUnits);
+const AcqUnitFilter = ({ resources, ...rest }) => {
+  const options = useMemo(() => getAcqUnitsOptions(resources), [resources]);
```

Comment:
`resources` - is  a quite general object, it's better to stick for specific part of it.

Reply:
Updated